data/net/minecraft/client/multiplayer/chat/ChatListener.mapping

```diff
@@ -2,70 +2,70 @@ CLASS net/minecraft/client/multiplayer/chat/ChatListener
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
 	METHOD evaluateTrustLevel (Lnet/minecraft/network/chat/PlayerChatMessage;Lnet/minecraft/network/chat/Component;Lnet/minecraft/client/multiplayer/PlayerInfo;Ljava/time/Instant;)Lnet/minecraft/client/multiplayer/chat/ChatTrustLevel;
-		ARG 1 message
-		ARG 2 decoratedServerContent
-		ARG 3 senderPlayerInfo
-		ARG 4 timeStamp
+		ARG 1 chatMessage
+		ARG 2 message
```

Comment:
I kinda like my arg 2 better. Also changing timestamp casing means checking the rest of the PR or using timeStamp. Whichever is higher in usage I guess.

Reply:
As for the the casing of timestamp, it would suggested to use `timestamp` instead of `timeStamp` in one of my PRs

Reply:
There are a few other cases in the merge commit that also were changed that I think should follow what you did here since you decided to change it to mine.

Reply:
This got undone